features/feature_case/asm/asm_ldmatrix.cu

```diff
@@ -0,0 +1,339 @@
+// ====------------- asm_ldmatrix.cu ------------- *- CUDA -* -------------===//
```

Comment:
This file is already in https://github.com/oneapi-src/SYCLomatic-test/pull/908

Reply:
Removed it